XamarinCommunityToolkit/Converters/BaseConverter.shared.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Globalization;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public abstract class BaseConverter<TFrom, TTo> : BaseConverterOneWay<TFrom, TTo>
```

Comment:
Should we consider making this internal to us? Do we expect people to implement their own? I guess they could... But that also means we're going to have to support this and keep it backwards compatible

Reply:
Hey @jfversluis , Yeah since it's all about sharing I thought this could be re-used by someone too ... I personally use it everywhere and am kind-of tired of Copy pasting it all the time :)
"keep it backwards compatible" ? With ? It should ensure more reliability for converters. One of the drawbacks I see with it is that you can't have a converter that takes multiple types of parameters and behaves differently depending on the type ...

Reply:
Gotcha @framinosona! It definitely is about sharing, but also about what we expect end-users to be using. I think one of the goals of this library is to keep it simple, that would mean people will probably only use inherited objects from this class. If I understand you correctly, you are using the `BaseConverter<TFrom, TTo>` directly as well? Why not a typed version of it?

> "keep it backwards compatible"

Look at it from the library perspective. Everything that we will add that is public, we commit to maintaining it and also keeping it backwards compatible. In example, if we want to make changes in `BaseConverter<TFrom, TTo>` and it's public, we do not control who has been using it. It can be all consumers of the Toolkit Nuget, or it could be none. But we will always have to assume there is someone since it's public. That means, if we want to change anything in this method, we would have to be careful to ideally not break the potential people that are using this method. Do you see what I'm trying to say? :)

Reply:
@jfversluis we can't make it internal, because we expose those in a `public` way. Maybe hiding from intellisense?